docs/how-to/installation/aws.txt

```diff
@@ -51,11 +51,11 @@ can be increased by submitting a request to the `AWS Support Center
 Installation
 ============
 
-``det deploy`` command line tool can be installed using ``pip``:
```

Comment:
I'm a little confused here - det-deploy executable is moving into the determined package, but this change implies that det deploy does not exist anymore?

Reply:
It implies the users only install `det` tool, which is what they'd use. New users shouldn't be aware `det` and `det deploy` subcommand used to be different things.

Reply:
Fair enough.